### PR TITLE
Select returns reminder template by invitation template used

### PR DESCRIPTION
**Pull request by DGajewska**
WATER-2518

* Add call for latest returnInvitation event with all related scheduled notifications which include a specific licence number
* Find appropriate template
* Use control template if no template found

### CODE REVIEW

File: src/lib/connectors/repos/events.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+const { Event, bookshelf } = require('../bookshelf');
+const queries = require('./queries/events');
 /**
  * creates a new event record in the database
  * @param {Object} event event pojo
  * @returns {Object} event pojo
  */
 const create = async (event) => {
-  const result = await Event.forge(event).save();
+  const result = await Event.forge().save(event);
```
Comment (Stephan-Hough) [inline src/lib/connectors/repos/events.js]: Interestingly both methods works. I followed the billing batch transactions approach to remain consistent. Did it not work the other way for you?

Comment (DGajewska) [reply src/lib/connectors/repos/events.js]: That's odd. The first way didn't work for the returns notifications feature, it only worked after the change I made

Comment (DGajewska) [reply src/lib/connectors/repos/events.js]: I had a second look at this after I figured out what was going wrong with this event and it does work the way you had it, so I've changed it back

File: src/modules/batch-notifications/lib/event-helpers.js [hunk 2]
```diff
@@ -63,13 +64,18 @@ const updateEventStatus = async (eventId, status, data = {}) => {
 const markAsProcessed = async (eventId, licenceNumbers, recipientCount) => {
   const ev = await evt.load(eventId);
```
[the remark this answers is not in the document]
Comment (Stephan-Hough) [reply src/modules/batch-notifications/lib/event-helpers.js]: agreed to do this later

File: src/modules/batch-notifications/lib/event-helpers.js [hunk 3]
```diff
@@ -44,11 +45,11 @@ const createEvent = async (issuer, config, options) => {
  */
 const updateEventStatus = async (eventId, status, data = {}) => {
   const ev = await evt.load(eventId);
```
[the remark this answers is not in the document]
Comment (Stephan-Hough) [reply src/modules/batch-notifications/lib/event-helpers.js]: agreed to do this later

**Pull request merged**